packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -14,18 +14,18 @@ Source0:    https://downloads.theforeman.org/%{name}/%{name}-%{version}%{?prerel
 BuildArch:  noarch
 
 Requires:   curl
-Requires:   %{?scl_prefix}puppet-agent >= 1.9.0
-Requires:   %{?scl_prefix}rubygem-kafo >= 1.0.5
+Requires:   %{?scl_prefix}puppet-agent >= 5.5.8
+Requires:   %{?scl_prefix}rubygem-kafo >= 3.0.0
 Requires:   foreman-selinux
 Requires:   %{?scl_prefix_ruby}ruby(release)
 Requires:   %{?scl_prefix}rubygem-highline
 
 BuildRequires: asciidoc
 BuildRequires: rubygem(rake)
 BuildRequires: %{?scl_prefix}puppet-agent >= 1.9.0
```

Comment:
I've kept this older since it can still build with older versions. Ideally we'd update this as well when our koji has a newer version.